<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCrsp</class>
 <widget class="QWidget" name="AddCrsp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Correspondences Manually</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_PushButtonGetCrspPt">
       <property name="text">
        <string>Add Correspondence Point</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QVTKWidget" name="m_SourceViewer"/>
     </item>
     <item>
      <widget class="QVTKWidget" name="m_TargetViewer"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
